services/py-api/src/service/participants_service.py

```diff
@@ -35,10 +37,45 @@ async def _create_participant_and_team_in_transaction(
             return participant
 
         return Ok((participant.ok_value, team.ok_value))
+    
+    async def _create_random_participant_in_transaction(
+            self, input_data: ParticipantRequestBody, session: Optional[AsyncIOMotorClientSession] = None
```

Comment:
Change the function signature to include Result[...]. Like it is done: 
```Python
async def _create_participant_and_team_in_transaction(
        self, input_data: ParticipantRequestBody, session: Optional[AsyncIOMotorClientSession] = None
    ) -> Result[Tuple[Participant, Team], DuplicateEmailError | DuplicateTeamNameError | Exception]:
```

Reply:
Fixed in 1bec98abe406eea6382a5453ac98f3e900d3ddd4